src/components/marker-chart/Canvas.js

```diff
@@ -266,28 +265,43 @@ class MarkerChartCanvasImpl extends React.PureComponent<Props> {
     h: CssPixels,
     uncutWidth: CssPixels,
     text: string,
-    backgroundColor: string = BLUE_40,
-    foregroundColor: string = 'white'
+    isHighlighted: boolean = false
   ) {
-    ctx.fillStyle = backgroundColor;
+    const { marginLeft } = this.props;
+    ctx.fillStyle = isHighlighted ? BLUE_70 : '#0A75E3'; // This color is in-between BLUE_50 and BLUE_60, and is contrasted enough for the white text.
+    ctx.strokeStyle = isHighlighted ? BLUE_80 : BLUE_60;
```

Comment:
I checked just in case the color contrasts are accessible and looks like they are. I slightly think that the previous version was a bit readable for me though. I think it's pretty good when it's not highlighted but when it's highlighted, I feel like it's harder to read. I wonder what you are thinking about this.

Reply:
Yeah I believe you're right for when it's highlighted: black on top of #003eaa isn't accessible.
I'll find a solution before landing, thanks for checking!